cmd/client/cmd/upload.go

```diff
@@ -46,9 +51,16 @@ func newUploadCmd() *cobra.Command {
 				return fmt.Errorf("upload file: get cli parameter 'file': %w", err)
 			}
 
-			data, err := os.ReadFile(filePath)
-			if err != nil {
-				return fmt.Errorf("upload file: read file from path: %w", err)
+			var data []byte
+
+			if strings.HasPrefix(filePath, "http") || strings.HasPrefix(filePath, "https") {
```

Comment:
include `://` - `http://` and `https://`. The folder name can start with `http`.